### PR TITLE
Add Car feature and tests and Update user guide

### PR DESCRIPTION
**Pull request by Meowloid**

### CODE REVIEW

File: src/test/java/seedu/address/logic/commands/AddCarCommandTest.java
Comment (limyuquan) [inline src/test/java/seedu/address/logic/commands/AddCarCommandTest.java:1]: Missing Test cases for AddCarCommandParserTest

**Review by limyuquan — CHANGES_REQUESTED** [review]
Everything else looks good, just missing AddCarCommandParserTest test cases

File: src/test/java/seedu/address/logic/commands/AddCarCommandTest.java
Comment (Meowloid) [reply src/test/java/seedu/address/logic/commands/AddCarCommandTest.java:1]: Done

**Review by limyuquan — APPROVED** [review]
lgtm

**Pull request merged**